Forge/src/main/java/com/blamejared/crafttweaker/CraftTweakerForge.java

```diff
@@ -15,11 +20,27 @@ public CraftTweakerForge() {
         
         PacketHandler.init();
         FMLJavaModLoadingContext.get().getModEventBus().addListener(this::setup);
+        MinecraftForge.EVENT_BUS.addListener(this::onCustomBracketParserRegistration);
+        MinecraftForge.EVENT_BUS.addListener(this::onLegacyBracketParserRegistration);
     }
     
     private void setup(final FMLCommonSetupEvent event) {
         
         CraftTweakerCommon.loadInitScripts();
     }
     
+    private void onCustomBracketParserRegistration(final CTRegisterCustomBepEvent event) {
+        
+        if(CraftTweakerConstants.MOD_ID.equals(event.getCurrentLoader())) {
+            // Fire the old event for compatibility: this is only required for the CrT loader
+            Services.EVENT.fireRegisterBEPEvent(new WrappingBracketParser(event::registerBep));
```

Comment:
What will happen here for a mod that implements both event listeners and adds a BEP in each of them with the same name?
Unlikely (since few mods even have custom event listeners) but just so that we talked about it ^^